### DISCUSSION TITLE
Maximise one view to full screen on dual view

### DISCUSSION THREAD

**Discussion by cyriltovena** [grafana/grafana#51223] [Explore]
I love the double view in explore, I used it very often, specially when I'm reading logs and then suddenly looking at traces.

<img width="2547" alt="image" src="https://user-images.githubusercontent.com/1053421/174961194-6fc6063e-6419-4a54-a513-c574a87dc77f.png">

One thing though that is really frustrating is that if I want to focus on a view like the trace view, my only option is to actually close the logs view. But I don't want to close it because I might still searching or exploring more traces.

My suggestion is here is to allow the same as in visual studio, which collapse vertically the view I'm not focusing see below:

https://user-images.githubusercontent.com/1053421/174961860-b6133279-e15f-45f7-92ef-79e7c97a93c4.mov

/cc @davkal

**Comment by vlad-diachenko** [comment]
I like your proposition. Also, I would love to see a button to open some part of the view(logs or traces) in a new tab to investigate only traces or only logs...

**Comment by Elfo404** [comment]
This has been implemented and should be available in Grafana 9.4 🎉 

https://user-images.githubusercontent.com/1170767/211858841-ba40d8d9-b000-4255-bc26-1a116bd4b786.mov
